extensions/amp-analytics/0.1/vendors.js

```diff
@@ -54,6 +54,7 @@ import {GOOGLEADWORDS_CONFIG} from './vendors/googleadwords';
 import {GOOGLEANALYTICS_CONFIG} from './vendors/googleanalytics';
 import {GTAG_CONFIG} from './vendors/gtag';
 import {IBEATANALYTICS_CONFIG} from './vendors/ibeatanalytics';
+import {INFONLINE_ANONYMOUS_CONFIG} from './vendors/infonline_anonymous';
```

Comment:
We recent refactored this file, you no longer need to make any change to this file : )